File: 2397.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){
    int a,b,c,l1,l2,l3;
    cin>>l1>>l2>>l3;
    if(l1>l2 && l1>l3){
        a = l1;
        b = max(l2,l3);
        c = min(l2,l3);
    }
    else if(l2>l3 && l2>l1){
        a = l2;
        b = max(l1,l3);
        c = min(l1,l3);
    }
    else{
        a = l3;
        b = max(l2,l1);
        c = min(l2,l1);
    }
    if(abs(a-b)<c && c<(a+b) && abs(c-a)<b && (c+a)>b && abs(c-b)<a && (c+b)>a){
        if(a*a==b*b + c*c) cout<<"r"<<endl;
        else if(a*a > b*b + c*c) cout<<"o"<<endl;
        else cout<<"a"<<endl;
    }
    else cout<<"n"<<endl;
}

File: 2397.py
def main():
    tex = input()
    lista = tex.split()
    l1 = int(lista[0])
    l2 = int(lista[1])
    l3 = int(lista[2])
    if(l1>l2 and l1>l3):
        A = l1
        B = max(l2,l3)
        C = min(l2,l3)
    elif(l2>l1 and l2>l3):
        A = l2
        B = max(l1,l3)
        C = min(l1,l3)
    else:
        A = l3
        B = max(l1,l2)
        C = min(l1,l2)
    if(abs(A-B)<C and C<(A+B) and abs(C-A)<B and (C+A)>B and abs(C-B)<A and (C+B)>A):
        if(A**2==B**2 + C**2):
            print("r")
        elif(A**2 > B**2 + C**2):
            print("o")
        else:
            print("a")
    else:
        print("n")
        
main()